core/src/main/java/org/stellar/anchor/util/SepLanguageHelper.java

```diff
@@ -0,0 +1,105 @@
+package org.stellar.anchor.util;
+
+import static org.stellar.anchor.util.Log.errorEx;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.stellar.anchor.api.exception.SepValidationException;
+import org.stellar.anchor.config.AppConfig;
+
+public class SepLanguageHelper {
+  static Map<String, Iso3316Language> fallbackLangs = null;
+
+  public static String validateLanguage(AppConfig appConfig, String lang) {
+    if (fallbackLangs == null) {
+      fallbackLangs = prepareFallbackLanguages(appConfig);
+    }
+
+    if (lang != null) {
+      List<String> languages = appConfig.getLanguages();
+      if (languages != null && languages.size() > 0) {
+        if (languages.stream().noneMatch(l -> l.equalsIgnoreCase(lang))) {
+          return getFallbackLanguage(lang);
+        }
+      }
+      return lang;
+    } else {
+      return getFallbackLanguage(lang);
+    }
+  }
+
+  public static void reset() {
+    fallbackLangs = null;
+  }
+
+  static Map<String, Iso3316Language> prepareFallbackLanguages(AppConfig appConfig) {
+    Map<String, Iso3316Language> fallbacks = new HashMap<>();
+    appConfig
+        .getLanguages()
+        .forEach(
+            lang -> {
+              try {
+                Iso3316Language isoLang = Iso3316Language.of(lang);
+                fallbacks.putIfAbsent(isoLang.getLangKey(), isoLang);
+              } catch (SepValidationException ex) {
+                errorEx(ex);
+              }
+            });
+    return fallbacks;
+  }
```

Comment:
Hey, I'm a bit confused by your usage of the word fallback. What you are calling fallback languages are actually the configured languages found in the config files. The word fallback refers to "an alternative plan". So the `LANGUAGE_ONLY_DEFAULT`, for instance is what we use as a fallback, rather than the languages from AppConfig.

Reply:
The word `fallback` is used in the[ slack discussion:](https://stellarfoundation.slack.com/archives/CHVFR99UJ/p1653323769451659) 

If no exact language is found, it will first fall back to a alternative language that is pre-defined. If no alternative language is found, it will fall back to a default. (We should make this configurable in the following PR).